common/src/main/java/org/apache/rocketmq/common/message/Message.java

```diff
@@ -210,7 +210,7 @@ public String toString() {
             "topic='" + topic + '\'' +
             ", flag=" + flag +
             ", properties=" + properties +
-            ", body=" + Arrays.toString(body) +
+            ", body=" + (body != null ? new String(body, StandardCharsets.UTF_8) : "") +
```

Comment:
The message body may be very large and compressed or serialized. It's better not to print the message body in `toString` function.

Reply:
> The message body may be very large and compressed or serialized. It's better not to print the message body in `toString` function.

Wasn't it also printed before this modification? Do you mean to delete this directly

Reply:
IMO, user can choose if to print message body,

so we can provide a readable format when users need to print message body

Reply:
> IMO, user can choose if to print message body,
> 
> so we can provide a readable format when users need to print message body

+1. Maybe we can check the size of message body first and only print the message body if the size is small.

Reply:
> > The message body may be very large and compressed or serialized. It's better not to print the message body in `toString` function.
> 
> Wasn't it also printed before this modification? Do you mean to delete this directly

Maybe print body size instead.

Reply:
Recommended print body size when body size more than 3000,otherwise, it is impossible to distinguish whether the body is empty just by looking at the log

Reply:
Do not print body is better, because the serialize of body is unknown.